### DISCUSSION TITLE
Hide taskbar icon

### DISCUSSION THREAD

**Discussion by ShinoNia** [r0x0r/pywebview#1066] [Q&A] [answered]
Is there any way to hide the icon created by a PyWebView window on Windows 10 taskbar?

**Comment by r0x0r** [comment, marked as answer]
Nope, this is not implemented

**Reply by ShinoNia** [reply]
Thanks for letting me know. May I ask is there any chance this will be possible in the future?

**Reply by r0x0r** [reply]
Sure, if somebody wants to give it a go, I will gladly accept a PR.

**Reply by ShinoNia** [reply]
This can be achieved by adding "self.ShowInTaskbar = False" to class BrowserForm in winforms.py
I don't know how to add it as an argument for create_window tho
